src/v/config/configuration.cc

```diff
@@ -825,6 +826,20 @@ configuration::configuration()
        .visibility = visibility::user},
       {},
       validate_connection_rate)
+  , kafka_rpc_server_tcp_recv_buf(
+      *this,
+      "kafka_rpc_server_tcp_recv_buf",
+      "Kafka server TCP receive buffer size in bytes.",
+      {.example = "65536"},
+      std::nullopt,
+      {.min = 32_KiB, .align = 4_KiB})
```

Comment:
Just checked, and yes, if you set stupidly high values the kernel just clamps you.

Reply:
ahh makes sense! I figured seastar was doing something (and it might if we tried to use dpdk and user space networking I wonder). anyway, we don't use that right now.